maven_plugin/src/test/resources-its/it/IntegrationTestsIT/environmentalCheckShouldFail/lockfile.json

```diff
@@ -0,0 +1,31 @@
+{
+  "artifactId": "environmental-check-should-fail",
+  "groupId": "com.mycompany.app",
+  "version": "1",
+  "pom": {
+    "path": "pom.xml",
+    "checksumAlgorithm": "SHA-256",
+    "checksum": "92f717596d6b518dcd79e5fa730fa2b72c81daaaa8d3246e049808bdc830d31f"
+  },
+  "lockFileVersion": 1,
+  "dependencies": [],
+  "mavenPlugins": [],
+  "metaData": {
+    "environment": {
+      "osName": "TamperedOS",
```

Comment:
I believe this would take the value `Linux`, `MacOS`, or `Windows` based on the environment? Since it is `TamperedOS`, there will always be a mismatch.